processor/metricstransformprocessor/metrics_transform_processor_testcases_test.go

```diff
@@ -74,13 +76,44 @@ var (
 					setDataType(metricspb.MetricDescriptor_GAUGE_DOUBLE).build(),
 			},
 		},
+		{
+			name: "metric_names_update_chained",
+			transforms: []internalTransform{
+				{
+					MetricIncludeFilter: internalFilterRegexp{include: regexp.MustCompile("^(metric)(?P<namedsubmatch>[12])$")},
+					Action:              Update,
+					NewName:             "new/$1/$namedsubmatch",
```

Comment:
Looks really cool! Tell people about this in the readme, looks like it's missing from there.